packages/rancher-gatekeeper/package.yaml

```diff
@@ -1,5 +1,5 @@
-url: https://open-policy-agent.github.io/gatekeeper/charts/gatekeeper-3.3.0.tgz
-packageVersion: 1
+url: https://open-policy-agent.github.io/gatekeeper/charts/gatekeeper-3.4.0.tgz
+packageVersion: 2
```

Comment:
If you are updating the major version, you can leave the packageVersion as `1`.

You only need to bump the packageVersion to avoid a conflict with an already released chart.

Reply:
Yes, I meant minor version 😅